Updated Package.php to add media folder to zip

shamsbd71 opened the pull request:
Component media folder missed to push in zip file. this patch will add that.

shamsbd71 committed:
Updated Package.php to add media folder to zip

yvesh commented:
Hi @shamsbd71, can you add a check if the media folder exists? 

We should do that for all folders there.. as copyDir throws a TaskException. But /media is the most important as for example com_weblinks does not have one.

Thanks for your contribution 👍

shamsbd71 committed:
Package.php updated to check if media folder exist

as all the component will not have a media folder, checking added as @yvesh requested.
thanks to @yvesh

shamsbd71 commented:
:) 
@yvesh I have added the check. 
Thanks

yvesh commented:
Works as expected 👍 Thank you!
